app/models/work.rb

```diff
@@ -11,11 +11,9 @@ class Work < ApplicationRecord
   has_many :related_works, dependent: :destroy
   has_many_attached :files
 
-  accepts_nested_attributes_for :contributors, allow_destroy: true, reject_if: :last_name_blank
-
   validates :abstract, :access, :citation, :state, :subtype, :title, presence: true
   validates :contact_email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }
-  validates :created_edtf, presence: true, edtf: true
+  validates :created_edtf, :published_edtf, edtf: true
```

Comment:
Does this mean we're no longer validating that `created_edtf` is required? Or did that validation move up into the form?

Reply:
The designs changed and created date is no longer required.

Reply:
@jcoyne Thank you!